ethereumj-core/src/main/resources/ethereumj.conf

```diff
@@ -234,10 +235,16 @@ cache {
     # the size of header queue cache during import in MBytes
     headerQueueSize = 8
 
+    # the size of read cache maintaining prune entries, in MBytes
+    # 8 MB of that cache should be enough to maintain a 256-blocks window
+    # window size is defined by database.prune.maxDepth
+    pruneCacheSize = 8
+
     # maximum size (in Mb) the state bloom fiter can grow up to
     # when reaching this threshold the bloom filter
     # is turned off forever
     # 128M can manage approx up to 50M of db entries
+    # deprecated, states source is no more backed with bloom filter
```

Comment:
may be capitalize DEPRECATED or smth like this, looks like a wall of text
same for the next "deprecated" note

Reply:
good idea 👍